### DISCUSSION TITLE
How to change the close button size inside the notifications system?

### DISCUSSION THREAD

**Discussion by miketdonahue** [mantinedev/mantine#5524] [Q&A] [answered]
Hi,

Fantastic library. Incredibly well done.

How do I change the close button size inside the [notifications system](https://mantine.dev/x/notifications/)?

I am having quite a bit of trouble as changing the default prop size for the `closeButtonProps` doesn't seem to behave the same way the individual close button does. My screenshot below shows a `--cb-icon-size` being injected and preventing the size of the SVG from changing even when the button size changes. This seems like an odd restriction.

<img width="1396" alt="image" src="https://github.com/mantinedev/mantine/assets/8016033/afc1c86c-b0ff-4306-abf3-8eb66a28d885">

Thanks!

**Comment by rtivital** [comment]
Use `defaultProps` for Notification component and pass props down to the close button with `closeButtonProps`
https://mantine.dev/theming/default-props/

**Reply by miketdonahue** [reply]
Hi @rtivital - Yes, that works, but the size of the SVG, the "X", does not change. In my screenshot, you can see the button indeed did get larger (I set it to `size="xl"` via the defaultProps). The problem seems to be the SVG size isn't getting larger with the button size. This seems to be an issue only inside the Notification component. The `CloseButton` on it's own works fine.

<img width="459" alt="image" src="https://github.com/mantinedev/mantine/assets/8016033/01dc39c5-eb31-48bb-a8d3-f92c9830df3e">

Toggling off the `--cb-icon-size` CSS variable seems to work. This seems to be the culprit.

<img width="1351" alt="image" src="https://github.com/mantinedev/mantine/assets/8016033/2fac13c6-867c-40e4-9f82-e8b51507d279">

<img width="455" alt="image" src="https://github.com/mantinedev/mantine/assets/8016033/001d2000-64da-4fca-a982-d63f9c9c8204">

**Reply by rtivital** [reply, marked as answer]
Have you tried `closeButtonProps={{ iconSize: 40 }}`?

**Reply by miketdonahue** [reply]
That did the trick. Sorry, I missed that in the `CloseButton` props. Really appreciate it!
